crates/sage-cli/src/input.rs

```diff
@@ -283,6 +288,21 @@ impl Input {
             None => CloudPath::Local(std::env::current_dir()?),
         };
 
+        let score_type = match self.score_type {
```

Comment:
See above comment, delete this code and let Serde handle deserialization.

Reply:
Removed custom string parsing, should now be solved by serde (nice feature, did not know it could do that).